src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2352,15 +2411,35 @@ public static String getRoutedSitePinFromPhysicalPin(Cell cell, Net net, String
      * @since 2023.1.2
      */
     public static List<String> getAllRoutedSitePinsFromPhysicalPin(Cell cell, Net net, String belPinName) {
+        return getAllRoutedSitePinsFromPhysicalPin(cell, net, belPinName, null);
+    }
+
+    /**
+     * Gets all site pins that are currently routed to the specified cell pin.  If
+     * the site instance is not routed, it will return null.
+     * @param cell The cell with the pin of interest.
+     * @param net The physical net to which this pin belongs
+     * @param belPinName The physical pin name of the cell
+     * @return A list of site pin names on the cell's site to which the pin is routed.
+     * @since 2023.1.2
```

Comment:
Probably update the Javadoc here as well, but the map description can be brief.  Also, we only mandate use of the `@since` tag in the closed source portion, but since it was on the original, you might as well update it to `@since 2025.2.0`.